platform_plugin_forum_email_notifier/models.py

```diff
@@ -57,3 +51,27 @@ class Meta:
 
         ordering = ["-created"]
         unique_together = ["user", "course_id"]
+
+
+class ForumNotificationDigest(TimeStampedModel):
+    """
+    A model to store forum email notification digests for a user.
+
+    .. no_pii:
+    """
+
+    user = models.ForeignKey(
+        to=User,
+        on_delete=models.CASCADE,
+        related_name="forum_notification_digests",
+    )
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    threads_json = models.TextField()
```

Comment:
can we specify what this is in the model docstring?

Reply:
fixed